searx/templates/oscar/preferences.html

```diff
@@ -318,12 +318,12 @@ <h3>{{ _('Engines') }}</h3>
                                     <th scope="col">{{ _("Allow") }}</th>
                                     <th scope="col">{{ _("Engine name") }}</th>
                                     <th scope="col">{{ _("Shortcut") }}</th>
-                                    <th scope="col" style="width: 10rem">{{ _("Selected language") }}</th>
-                                    <th scope="col" style="width: 10rem">{{ _("SafeSearch") }}</th>
-                                    <th scope="col" style="width: 10rem">{{ _("Time range") }}</th>
+                                    <th scope="col">{{ _("Selected language") }}</th>
```

Comment:
`width: 10rem` is useful on on wide screen: it allows to keep the column small, and give space to the other columns.

Reply:
Fixed in https://github.com/searxng/searxng/pull/295/commits/222031e975bc428b77a00878776744f3de884ca9